packages/@guardian/source-react-components-development-kitchen/index.tsx

```diff
@@ -1,3 +1,4 @@
+export { AlertSwitch } from './components/alert-switch';
```

Comment:
It might be worth exporting the types too

```suggestion
export { AlertSwitch } from './components/alert-switch';
export type { AlertSwitchProps } from './components/alert-switch';
```